<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorDecorator</class>
 <widget class="QWidget" name="GeneratorDecorator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneratorDecorator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxDecorator">
     <property name="text">
      <string>decorator</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditDecorator"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxComponent">
     <property name="text">
      <string>component</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditComponent"/>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxInterface">
     <property name="text">
      <string>interface</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonInterface">
     <property name="text">
      <string>define interface</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
